src/LessMsi.Cli/Program.cs

```diff
@@ -188,10 +192,23 @@ private static void PrintProgress(IAsyncResult result)
 
         private static void EnsureFileRooted(ref string sFileName)
         {
-            if (!Path.IsPathRooted(sFileName))
+            if (!IsPathRooted(sFileName))
             {
                 sFileName = Path.Combine(Directory.GetCurrentDirectory(), sFileName);
             }
         }
+
+        private static void EnsureAbsolutePath(ref string filePath)
+        {
+            if (!IsPathRooted(filePath))
+            {
+                filePath = Path.GetFullPath(filePath);
+            }
+        }
+
+        private static bool IsPathRooted(string filePath)

```

Comment:
I'd prefer we didn't add these additional lines here that only call one method and instead just call Path.IsPathRooted